Core.ApplicationServices/System/ItSystemService.cs

```diff
@@ -403,6 +403,11 @@ public bool CanCreateSystemWithName(int organizationId, string name)
             });
         }
 
+        public Result<ItSystem, OperationError> UpdateExternalUuid(int systemId, Guid? newExternalUuid)
+        {
+            return Mutate(systemId, system => system.ExternalUuid != newExternalUuid, system => system.ExternalUuid = newExternalUuid);
```

Comment:
What happens if newExternalUuid is null? - does this mean overwrite the old value or is it just a way of saying "no change"? After having created a new ItSystem and having set the externalUuid, how do we handle the situation where the other party no longer thinks there is an ItSystem?